### PR TITLE
Update SRUM.tkape

### PR DESCRIPTION
**Pull request by mark-hallman**
## Description

Added SOFTWARE hive and logs to this target as they are typically used in processing SRUM data.

## Checklist:
Please replace every instance of `[ ]` with `[X]`

- [X ] I have generated a unique GUID for my Target(s)/Module(s)  - Existing target so I didn't generate a new GUID.
- [X] I have placed the Target/Module in an appropriate subfolder in Targets or Modules. If one doesn't exist, I have either added it to the Misc folder or created a relevant subfolder **with justification**  YES.
- [X] I have verified that KAPE parses the Target successfully via kape.exe, using `--tlist`/`--mlist` and corrected any errors.  YES.

### CODE REVIEW

File: Targets/Windows/SRUM.tkape [hunk 1]
```diff
@@ -14,6 +14,26 @@ Targets:
...
+        Name: SOFTWARE registry hive
+        Category: Registry
+        Path: C:\Windows\System32\config\
+        FileMask: SOFTWARE
+    -
+        Name: SOFTWARE registry hive
+        Category: Registry
+        Path: C:\Windows.old\Windows\System32\config\
+        FileMask: SOFTWARE   
+
```
Comment (rathbuna) [inline Targets/Windows/SRUM.tkape:27]: Did you --tlist this? It will probably shoot an error here by not having the -

**Review by rathbuna — CHANGES_REQUESTED** [review]

**Comment by EricZimmerman** [comment]
Also make sure you are getting log files.

File: Targets/Windows/SRUM.tkape (hunk 1 above)
Comment (mark-hallman) [reply Targets/Windows/SRUM.tkape:27]: rathbuna, yeah,  I did.  

I see the error that you are pointing out and will resubmit the pull request.  i have attached a few screen shots.  --tlist does not complain and the target produced the desired results.  weird.

thanks for checking and thanks for all your contributions to these projects.

**Comment by mark-hallman** [comment]
Eric,  the target does grab the log files.  Thanks for checking.

File: Targets/Windows/SRUM.tkape (hunk 1 above)
Comment (rathbuna) [reply Targets/Windows/SRUM.tkape:27]: > 
> 
> rathbuna, yeah, I did.
> 
> I see the error that you are pointing out and will resubmit the pull request. i have attached a few screen shots. --tlist does not complain and the target produced the desired results. weird.
> 
> thanks for checking and thanks for all your contributions to these projects.
> 
> 
> 
> 
> 
> 

Interesting! You shouldn't need to resubmit the PR. If you just do another commit to your fork with the added -, it should automatically include that change within this PR before we merge. That's typically how it's worked for me. 

@EricZimmerman do you have any insight on the missing - and if it'll be an issue/why it's not complaining about it?

Comment (mark-hallman) [reply Targets/Windows/SRUM.tkape:27]: Oh crap,  two versions of the target on the machine.   sorry for being an idiot.

**Review by mark-hallman — COMMENTED** [review]
I think that I have submitted the updated pull request, the one with the hyphen added.  I have not used the "change requested" feature on a pull request.  Still learning.

**Comment by rathbuna** [comment]
> 
> 
> I think that I have submitted the updated pull request, the one with the hyphen added. I have not used the "change requested" feature on a pull request. Still learning.

You nailed it. Thanks for fixing that!

**Pull request merged**